src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/PropertySetAnalysis/PropertySetAnalysis.cs

```diff
@@ -202,87 +262,6 @@ public static HazardousUsageEvaluationResult MergeHazardousUsageEvaluationResult
             }
         }
 
-        /// <summary>
-        /// Enumerates literal values to map to a property set abstract value.
-        /// </summary>
-        /// <param name="valueContentAbstractValue">Abstract value containing the literal values to examine.</param>
-        /// <param name="badLiteralValuePredicate">Predicate function to determine if a literal value is bad.</param>
-        /// <returns>Mapped kind.</returns>
-        /// <remarks>
-        /// Null is not handled by this.  Look at the <see cref="PointsToAbstractValue"/> if you need to treat null as bad.
-        /// 
-        /// All literal values are bad => Flagged
-        /// Some but not all literal are bad => MaybeFlagged
-        /// All literal values are known and none are bad => Unflagged
-        /// Otherwise => Unknown
-        /// </remarks>
-        public static PropertySetAbstractValueKind EvaluateLiteralValues(
```

Comment:
I am presuming this just moves the existing code to a new file?

Reply:
Yep.